.github/workflows/community_test_suite.yml

```diff
@@ -15,8 +15,8 @@ on:
         type: string
   push:
     branches: [ main, dev ]
-  # Removed pull_request trigger to avoid testing everything on every PR
-  # Individual adapter tests will run based on their specific path triggers
+  pull_request:
+    branches: [ main, dev ]
 
```

Comment:
I've temporarily added this to run the community test suite on this PR for testing purposes.

After approval, I'll revert this